template/src/validateInvocation.ts

```diff
@@ -0,0 +1,14 @@
+import { IntegrationExecutionContext } from '@jupiterone/integration-sdk';
+
+export default function validateInvocation(
+  context: IntegrationExecutionContext,
+) {
+  context.logger.info(
+    {
+      instance: context.instance,
+    },
+    'Validating integration config...',
+  );
+
+  context.logger.info('Integration instance is valid!');
+}
```

Comment:
We may want to include some guidance on what this function needs to do when invocation is not valid. I think it would be as simple as:
```
if (valid) {
  context.logger.info('Integration instance is valid!');
} else {
  throw new Error("Failed to authenticate with provided credentials!")
}
```